trpcultivate_phenotypes/src/Plugin/TripalImporter/TripalCultivatePhenotypesTraitsImporter.php

```diff
@@ -1472,24 +1472,45 @@ public function postRun() {
   }
 
   /**
-   * {@inheritdoc}
+   * Provide a more informative description about the template file.
+   *
+   * Class TripalImporterBase is the parent class of this method and additional
+   * documentation is available in reference link below.
+   *
+   * NOTE: This method supports full HTML markup output.
+   *
+   * All relevant information relating to expected column headers and usage
+   * notes are laid out using the theme 'importer_header'. A template geneartor
+   * service is utilized to provide a downloadable file template, pre-configured
+   * to contain all headers required.
+   *
+   * @see Tripal\src\TripalImporter\TripalImporterBase.php
```

Comment:
This should be the namespaced class and should point to the particular method.

```suggestion
   * @see Drupal\tripal\TripalImporter\TripalImporterBase::describeUploadFileFormat()
```